Terasology/LightAndShadowResources
# feat: game over screen skin rework

## Purpose

This PR is about modifications according to the design of the Game Over Screen.

## Changes made

* Added a `family` part at the elements within the UI file at the 

## How to test

1. Check https://github.com/Terasology/LightAndShadow/pull/146
2. Start a LAS game
3. Choose a team
4. Open the developer tab and gain the enemy team's by writting `give` and `redflag` or `blackflag`
5. Score as many points as you need to finish up the game
6. The Game Over Screen will pop up when you score the last point

_Note:_ You can go to `LAUtils.java` and change `GOAL_SCORE` variable from `5` to `1`, so you need to score 1 point to end the game.

## Current Version

![image](https://user-images.githubusercontent.com/48293545/90937107-49adf100-e40f-11ea-89f3-5b9c4ab1d87c.png)
